conf/messages

```diff
@@ -29,7 +29,7 @@ error.utr.blank= Enter your business''s Unique Taxpayer Reference (UTR)
 error.utr.invalid.format = Unique Taxpayer Reference (UTR) must only include numbers and spaces
 error.utr.invalid.length = Unique Taxpayer Reference (UTR) must be 10 numbers
 error.business-name.empty = Enter your business name
-error.business-name.invalid = Business name must only include letters a to z, numbers, commas, full stops, apostrophes, hyphens, ampersands, forward and backward slashes, and spaces
+error.business-name.invalid = Business name must only include letters a to z, numbers, commas, full stops, apostrophes, hyphens, forward and backward slashes, and spaces
```

Comment:
Actually, are backwards slashes allowed?

Reply:
There's a little confusion on ES8. It may or may not allow forward and/or backslashes. Probably worth giving it a try with those in.